test/src/receiver/main.cpp

```diff
@@ -28,102 +28,148 @@ using TIME = NDTime;
 
 
 /***** SETING INPUT PORTS FOR COUPLEDs *****/
-struct inp : public cadmium::in_port<Message_t>{};
+struct inp : public cadmium::in_port<Message_t>{
+
+};
 
 /***** SETING OUTPUT PORTS FOR COUPLEDs *****/
-struct outp : public cadmium::out_port<Message_t>{};
+struct outp : public cadmium::out_port<Message_t>{
```

Comment:
1) To have uniformity give a space between < and { 
i.e., struct outp : public cadmium::out_port<Message_t> {

2) it is not required to break the curly brackets but if you're breaking I would suggest to remove the empty line.

Reply:
Solved